client/pages/privacy-policy/privacy-policy.html

```diff
@@ -1,120 +1,680 @@
-      </div>
-    </section>
-  </div>
+    <div id="ah-privacy-policy-page">
+        <section>
+            <div class="ah-container">
+                <div class="page-title">Privacy Policy</div>
+
+                <p> We at AdmitHub, Inc. (collectively with any subsidiaries, "AdmitHub", “we”, or “us”) have created
+                    this Privacy Policy to describe our practices regarding information we collect through our web
+                    sites, including www.admithub.com, mobile features, applications and any other interactive features
+                    or services owned or controlled by AdmitHub that post a link to this Privacy Policy (each, a
+                    "Service" and collectively, the "Services"), as well as any information we collect offline and
+                    combine in our databases. Certain features discussed in this Privacy Policy may not be offered on
+                    each Service at any particular time. Note that we combine the information we collect from you
+                    through all of our websites, mobile applications, and other Services.</p>
+
+                <p>Note about Children: Our Services are not intended for, marketed to or available to individuals
+                    younger than the age of 13, and individuals younger than the age of 13 should not submit personal
+                    information to us or use any Service.</p>
+
+                <p>
+                    <strong>
+
+                        <h5>SUMMARY</h5>
+                        WE FULLY DESCRIBE OUR PRIVACY PRACTICES BELOW IN THIS PRIVACY POLICY. THIS SUMMARY PROVIDES AN
+                        OVERVIEW OF SOME IMPORTANT INFORMATION REGARDING OUR USE AND SHARING OF YOUR INFORMATION, AND
+                        THIRD PARTIES WHO MAY SERVE ADVERTISEMENTS AND WHO MAY SET COOKIES OR WEB BEACONS OR SIMILAR
+                        TRACKING TECHNOLOGIES WHEN YOU USE THE SERVICES, PLEASE READ THE ENTIRE PRIVACY POLICY VERY
+                        CAREFULLY. BY USING ANY SERVICE, YOU AGREE TO BE BOUND BY THIS PRIVACY POLICY IN ITS ENTIRETY.
+                    </strong>
+                </p>
+
+                <p>
+
+                    We primarily use the information we collect when you use the Services in connection with your
+                    relationship with AdmitHub, your use of the Services, and for sending you information from us or on
+                    behalf of certain third parties. This includes linking you to other members of the AdmitHub
+                    community. It also includes providing offers to you based on information you provide or we collect,
+                    or your use of the Services. Please review the "INFORMATION COLLECTION" and "HOW WE USE YOUR
```

Comment:
I think these references to parts of the document should be internal links, like `<a href="#collection"><strong>Information Collection</strong></a>`